docs/releases/v2.0.1.md

```diff
@@ -0,0 +1,36 @@
+# Monitoring Core Module Release 2.0.0
+
+Welcome to the latest release of the `monitoring` core module of the [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution), maintained by team SIGHUP.
+
+This is a patch release including the following change:
+- added `GOMAXPROCS=1` envrionment variable to `node-exporter` to avoid throttling (see #106).
```

Comment:
```suggestion
This is a patch release including the following changes:
- added `GOMAXPROCS=1` envrionment variable to `node-exporter` to avoid throttling (see [#106](https://github.com/sighupio/fury-kubernetes-monitoring/issues/106)).
- aligned Service port name for `kube-controller-manager` and `kube-schedule` with the one expected by the ServiceMonitors (see [#109](https://github.com/sighupio/fury-kubernetes-monitoring/issues/109) )
```